apps/storybook/src/Selection.stories.tsx

```diff
@@ -24,6 +30,45 @@ function vectorToStr(vec: Vector2) {
   return `(${format('.2f')(vec.x)}, ${format('.2f')(vec.y)})`;
 }
 
+const X = range(-5, 5, 0.01);
+const Y = X.map((x) => Math.exp((-x * x) / 0.2));
+
+function GaussianCurve() {
+  return <DataCurve abscissas={X} ordinates={Y} color="blue" />;
+}
+
+const modifierKeyArgType = {
+  control: { type: 'inline-radio' },
+  options: ['Alt', 'Control', 'Shift'],
+};
+
+const SelectionArgType = {
+  control: { type: 'inline-radio' },
+  options: ['line', 'rectangle'],
+};
```

Comment:
I put the arg types in constants so that `sonarjs` would stop complaining about the repetition of `inline-radio`

Reply:
You could also just disable the rule in `stories` files in `apps/storybook/.eslintrc.js`